.gitignore

```diff
@@ -49,6 +49,7 @@ _ReSharper.Caches/
 
 # Cores
 *.core
```

Comment:
I would rather not have cores in gitignore. It makes it seem like it crashes all the time. Let's just manually make sure we never commit one, and if we do, we can just delete it later.